Patches/Moyo from the depth/ThingDefs_Misc/Weapons.xml

```diff
@@ -440,7 +440,7 @@
 				</Properties>
 				
 				<AmmoUser>
-					<magazineSize>7</magazineSize>
+					<magazineSize>30</magazineSize>

```

Comment:
More a personal opinion than a balance issue, but this magazine almost seems so large as to be kind of not that beneficial for a semi-auto pistol, even on with slightly higher than average range. Depending on what your feelings on this weapon are, maybe a 2-round burst would be useful?

Reply:
30 was a guesstimate of how many rounds reasonably go in a nailgun, since 7 seemed likea bit few.
 
 Tbh, while I certainly think that a burst fire mode would help, I don't think it's reasonably required. The nailgun is reasonably more of a low bulk/mass holdout weapon, and between the range and better than average penetration I think it serves a sufficent role as a earlygame sidearm/backup type weapon.